### ISSUE TITLE
Hint Pop-up for Single Digit to be removed

### ISSUE THREAD

**Issue by KevinGeejo** [ankitkat042/ThalaForAReason#5]
The Hint Pop-up for input of a single digit should not pop up, as there is nothing to add.

    
    if (sum === 7) {
        showCongratulations(tabName);
    } else {
        showAlert('Try again!', tabName);
        showAlert('hint: addition of numbers', tabName);
    }

**Comment by ankitkat042** [comment]
Yeah, I did it quite hastily. My initial thought was that it would show 'Try again \n Hint: addition of numbers,' but only the second alert got displayed. Can you add an exception for single numbers to not show the addition hint and only display 'Try again'?

**Comment by KevinGeejo** [comment]
Okay I'll add it.
